GMnetENGINE.gmx/GMnetENGINE.project.gmx

```diff
@@ -116,6 +117,7 @@
           <script>scripts\htme_commitData.gml</script>
         </scripts>
         <script>scripts\htme_getPlayerNumber.gml</script>
+        <script>scripts\htme_disconnectNow.gml</script>

```

Comment:
Use in game to disconnect a server or client. Just call the script and it will handle the rest. You can also add
`if htme_disconnectNow() {
   do some cleaning yourself
   room_goto(my_menu)}`